src/sentry/api/bases/organization_events.py

```diff
@@ -220,13 +220,20 @@ def handle_results_with_meta(
         organization: Organization,
         project_ids: Sequence[int],
         results: Dict[str, Any],
+        standard_meta: Optional[bool] = False,
     ) -> Dict[str, Any]:
         with sentry_sdk.start_span(op="discover.endpoint", description="base.handle_results"):
             data = self.handle_data(request, organization, project_ids, results.get("data"))
             meta = results.get("meta", {})
 
+            if standard_meta:
+                isMetricsData = meta.pop("isMetricsData", False)
```

Comment:
Maybe we should start storing `isMetricsData` as a constant somewhere since we're using it quite frequently now